protocol-tests/src/samples/java/fixtures/bodystring/generated/EnumGetNotExpandable.java

```diff
@@ -11,10 +11,10 @@
 
 public class EnumGetNotExpandable {
     public static void main(String[] args) {
-        // BEGIN: fixtures.bodystring.generated.enumgetnotexpandable.enumgetnotexpandable
         EnumClient enumClient = new EnumClientBuilder().host("http://localhost:3000").buildClient();
+        // BEGIN:fixtures.bodystring.generated.enumgetnotexpandable.enumgetnotexpandable
```

Comment:
Adjust the location of the sample code.

If we automatically embed the sample into javadoc of client method, the initialization of the client should be self-evident (another potential issue is that client initialization code could be incorrect).